recipes/scine-utilities-python/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "utilities" %}
+{% set pkg_name = "scine-" + name + "-python" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/qcscine/utilities/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 3bcabd8f4270f0c1efd235a0d22506f4750b0ccffa99184d4362b95a1da7a1d4
+    fn: {{ name }}.tar.gz
+    patches:
+      - skip_library_build.patch
+  - url: https://github.com/qcscine/development-utils/archive/refs/tags/4.0.0.tar.gz
+    sha256: 54002c2082b6bb75672ec66bf9cf3935bbdf6b085ed9b4d7174cbdedb7c2275d
+    fn: dev.tar.gz
+    folder: dev
+    patches:
+      - pybind11_pin.patch
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  script:
+    - set -ex
+    - >
+       cmake
+       ${CMAKE_ARGS}
+       -B _build -G Ninja
+       -DPYTHON_EXECUTABLE={{ PYTHON }}
+       -DSCINE_MARCH=""
+       -DSCINE_SKIP_LIBRARY=ON
+       -DSCINE_BUILD_TESTS=OFF
+       -DSCINE_BUILD_PYTHON_BINDINGS=ON
+    - cmake --build _build
+    - cmake --install _build
+
+  run_exports:
+    - {{ pin_subpackage(pkg_name, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - pkg-config
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - boost-cpp
+    - eigen
+    - gtest
+    - gmock
+    - libblas
+    - pip
+    - pybind11
+    - pybind11-abi
+    - python
+    - scine-core
+    - scine-utilsos {{ version }}
+    - yaml-cpp
+  run:
+    - boost-cpp
+    - eigen
+    - numpy
+    - pybind11
+    - python
+    - scine-utilsos {{ version }}
+    - scipy
+    - yaml-cpp
```

Comment:
```bash
$ldd scine_utilities.cpython-37m-x86_64-linux-gnu.so 
	linux-vdso.so.1 (0x00007ffc659e6000)
	libutilsos.so => not found
	libcore.so => not found
	libyaml-cpp.so.0.6 => not found
	libblas.so.3 => /lib/x86_64-linux-gnu/libblas.so.3 (0x00007f7c62e30000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f7c62c4e000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f7c62afd000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f7c62ae2000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f7c628f0000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f7c630f1000)
```
Also, please take a look at these [overlinking warnings](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=482067&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1681) from conda-build. If you're only using the header-only libraries from pybind11 and boost, then they should be excluded from the run requirements.

Also, eigen is header only?

Reply:
Removed all unneeded run requirements